CONTRIBUTING.md

```diff
@@ -79,13 +83,28 @@ ansible-test units --docker -v --python 3.8
 ansible-test units --docker -v --python 3.8 tests/unit/plugins/modules/net_tools/test_nmcli.py
 ```
 
+### Integration tests
+
 The following commands show how to run integration tests:
 
+#### In docker
+
+Integration tests on docker require two parameters:
+- `image_name`: The name of the docker image. To get the list of supported docker images, run
+  `ansible-test integration --help` and look for _target docker images_.
+- `module_name`: The name of the module. For example, pacman in case of community.general.pacma
```

Comment:
```suggestion
- `test_name`: The name of the integration test. For modules, this equals the short name of the module; for example, `pacman` in case of `community.general.pacman`. For plugins, the plugin type is added before the plugin's short name, for example `callback_yaml` for the `community.general.yaml` callback.
```
(Probably should have line breaks.)